monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -29,7 +29,7 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 [IMPORTANT]
 ====
-In {product-title} 4.1 deployed on bare metal, Prometheus and Alertmanager cannot have persistent storage and are ephemeral only. For use cases other than bare metal, you must use dynamic provisioning to use persistent storage. In particular, do not use NFS for the monitoring stack.
+See xref:../../scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].
```

Comment:
s/.html/.adoc

Reply:
Good catch, implemented.

Reply:
Thanks! We'll just need the commits squashed into one, then I can merge and cherry-pick.

Reply:
Squashed. Thank you.